### PR TITLE
Trackers

### PR DESCRIPTION
**Pull request by sv01a**
newstudio.tv - new ru semi-private tracker 
rutracker - add series parser
rutor - global search without category + series parser

### CODE REVIEW

File: src/Jackett.Common/Definitions/newstudio.yml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      selector: a[href="/login.php?logout=1"]
+
+  search:
+    paths:
+      - path: "/tracker.php?nm={{ .Keywords}}"
+      - path: "/tracker.php?nm={{ .Keywords}}&start=50"
+      - path: "/tracker.php?nm={{ .Keywords}}&start=100"
+      - path: "/tracker.php?nm={{ .Keywords}}&start=150"
+    inputs:
+      nm: "{{ .Query.Keywords }}"
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/newstudio.yml]: if you have hard-coded nm= on the paths then the 'inputs' block is redundant

File: src/Jackett.Common/Definitions/newstudio.yml [hunk 2]
```diff
@@ -0,0 +1,103 @@
+﻿---
+  site: newstudio
+  name: Newstudio
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/newstudio.yml]: Please add NewStudio to the readme.md in the 'Supported Semi-Private Trackers' section

File: src/Jackett.Common/Definitions/rutor.yml [hunk 3]
```diff
@@ -51,18 +51,22 @@
   search:
     paths:
       # http://rutor.info/search/0/0/000/0/gotham%2004x01
-      - path: "{{ if .Keywords }}search/0/0/{{ .Config.method }}00/0/{{ .Keywords }}{{else}}top{{end}}"
+      - path: "{{ if .Query.Q }}search/{{ .Query.Q }}{{else}}top{{end}}"
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/rutor.yml]: if you are going to drop the config.method from the path then you might as well remove it from the settings block.

File: src/Jackett.Common/Definitions/newstudio.yml (hunk 2 above)
Comment (sv01a) [reply src/Jackett.Common/Definitions/newstudio.yml]: done

**Comment by garfield69** [comment]
Cheers

**Pull request merged**